fast_llm/engine/config_utils/checkpoint.py

```diff
@@ -0,0 +1,147 @@
+# TODO: Use packaging.version? (Safer but extra requirement)
```

Comment:
not sure where to ask this question:
where will we be configuring the layers-per-step option?

Reply:
That's a hack for the single process converter, it doesn't apply here because we already have the whole model loaded. I'm not entirely sure about memory usage but it should be OK because we reconstruct one layer at the time.